dataverse/orgsvc/C#-NETCore/xMultipleSamples/ElasticUtility.cs

```diff
@@ -0,0 +1,141 @@
+        /// <param name="tableSchemaName">The SchemaName of the table to create.</param>
+        public static void CreateExampleTable(ServiceClient serviceClient, string tableSchemaName)
+        {
+            JObject entityMetadataObject = new JObject()
+            {
+                { "SchemaName", tableSchemaName },
+                { "Description", new JObject()
+                    {
+                        { "@odata.type",  "Microsoft.Dynamics.CRM.Label" },
+                        { "LocalizedLabels", new JArray()
+                            {
+                                new JObject()
+                                {
+                                    {"@odata.type", "Microsoft.Dynamics.CRM.LocalizedLabel" },
+                                    { "Label", "A table for code samples." },
+                                    { "LanguageCode", 1033 }
+                                }
+                            }
+                        }
+                    }
+                },
+                { "DisplayCollectionName", new JObject()
+                    {
+                        { "@odata.type", "Microsoft.Dynamics.CRM.Label" },
+                        { "LocalizedLabels", new JArray()
+                            {
+                                new JObject()
+                                {
+                                    {"@odata.type", "Microsoft.Dynamics.CRM.LocalizedLabel" },
+                                    { "Label", "Examples" },
+                                    { "LanguageCode", 1033 }
+                                }
+                            }
+                        }
+                    }
+                },
+                { "DisplayName", new JObject()
+                    {
+                        { "@odata.type", "Microsoft.Dynamics.CRM.Label" },
+                        { "LocalizedLabels", new JArray()
+                            {
+                                new JObject()
+                                {
+                                    { "@odata.type", "Microsoft.Dynamics.CRM.LocalizedLabel" },
+                                    { "Label", "Example" },
+                                    { "LanguageCode", 1033}
+                                }
+                            }
+                        }
+                    }
+                },
+                { "OwnershipType", "UserOwned" },
+                { "TableType", "Elastic" },
+                { "IsActivity", false },
+                { "CanCreateCharts", new JObject()
+                    {
+                        { "Value", false },
+                        { "CanBeChanged", true },
+                        { "ManagedPropertyLogicalName", "cancreatecharts" }
+                    }
+                },
+                { "HasActivities", false },
+                { "HasNotes", false },
+                { "Attributes", new JArray()
```

Comment:
I don't see the `sample_Description` column added here like the others.

This sample supports another sample about writing plug-ins, so the tables and data need to be the same.

Reply:
this is only for creating the table, the other common attributes are getting created in utility itself